data/mods.json

```diff
@@ -494,5 +494,1461 @@
+      "no_medieval_items",
+      "no_migo_shooters",
+      "no_mutagen",
+      "no_npc_food",
+      "no_olg_guns",
+      "no_pipe_guns",
+      "no_religious_Texts",
+      "no_reviving_zombies",
+      "no_rivtech_guns",
+      "no_rural",
+      "no_scifi",
+      "no_survivor_armor",
+      "noctpk",
+      "novitamins",
+      "npc_traits",
+      "nuclear_ball",
+      "nw_pack",
+      "oa_additional_buildings_mod",
+      "oa_early_game_mutations_mod",
+      "offal-sausage",
+      "old_healing",
+      "overmap_special_fix",
+      "package_bionic_professions",
+      "pipe_revolver_mod",
+      "pistol_cross",
+      "pizza_delivery",
+      "pneumatic",
+      "pneumo_fun",
+      "porawep",
+      "portable_cooler",
+      "powder_disinfectant",
+      "pressure_weapon",
+      "primitiveplank",
+      "private_defense",
+      "psychiclysm",
+      "psychopath_morale",
+      "purple_drank",
+      "randomized_mall",
+      "realguns",
+      "realistic-bows",
+      "reapermod",
+      "recruitment_options",
+      "reproduction_mod",
+      "rf_food_keeper",
+      "rice",
+      "robot_automation",
+      "rolled_newspaper",
+      "rubber_band_sheath",
+      "ruralbiome",
+      "safe_portable_autodoc",
+      "sarcophagi_access",
+      "sci_turret",
+      "secronom",
+      "sees_player",
+      "sees_player_hitbutton",
+      "sees_player_less_attitude",
+      "sees_player_retro",
+      "service_weapon",
+      "seuzpack",
+      "shardsOtherStuff",
+      "shardstuff",
+      "shrub_mod",
+      "sierra_madre",
+      "simple_additions",
+      "sleepdeprivation",
+      "sly_beastfix",
+      "sly_mutation_improvements",
+      "sly_mutation_revamp",
+      "sonic",
+      "space_heater",
+      "speedydex",
+      "speedydexbuffs",
+      "stamina_pool_buff",
+      "stamina_regen_buff",
+      "stats_through_kills",
+      "stomach_offal",
+      "sunken_bridges",
+      "surv_crowbar",
+      "tamebugs",
+      "target_marking",
+      "tekitoumap",
+      "temp_effects_reduced",
+      "temp_effects_removed",
+      "terrainbasemod",
+      "thermal_ups",
+      "toki_sillyclasses",
+      "tool_bundle",
+      "trailerpark",
+      "tropicalpack",
+      "vamp_stuff",
+      "vamp_stuff+arcana",
+      "vorpal_weapons",
+      "wandering_masters",
+      "wastelands",
+      "water_towers",
+      "waxed_clothes",
+      "weaker-turrets",
+      "wet_morale_removed",
+      "whaleys_tallbuild",
+      "windoor",
+      "winning_dda",
+      "with_extra_mushrooms",
+      "xprofessions",
+      "xtal",
+      "zets_hair_ext_more_locations",
+      "zets_hair_ext_pk_rebalance",
+      "zets_hair_extensions_undead",
+      "zets_tweaks",
+      "eps_cbm"
+    ],
+    "name": "TheGoatGod's Mod Compilation",
+    "description": "Welcome to TheGoatGod's Mod Compilation. Currently contains over 275 mods, 5 tile sets, 40 NO Mods, and several sound packs. May need some editing to work after installation.",

```

Comment:
Welcome to TheGoatGod's Mod Compilation. Currently contains just under 400 mods, not including mainline mods, 14 maintained mods, 50 rebalance mods, 22 legacy mods, 2 tile sets(legacy working but lacks a lot of new items etc...), 49 NO Mods, 8 sound packs. It is currently, at 0.1.5, not entirely bug-free, but it is undergoing constant improvements, should soon be bug-free. Many changes are planned for the future, consolidating mods to reduce the amount of excess and cleaning up the code (even more!).